addons/website/views/snippets/s_picture.xml

```diff
@@ -8,10 +8,10 @@
             <p style="text-align: center;">Choose a vibrant image and write an inspiring paragraph about it.<br/> It does not have to be long, but it should reinforce your image.</p>
             <p style="text-align: center;"><br/></p>
             <div class="row s_nb_column_fixed">
-                <div class="col-lg-10 offset-lg-1 pb24" style="text-align: center;">
+                <div class="col-lg-12 pb24">
                     <figure class="figure">
-                        <img src="/web/image/website.s_picture_default_image" class="figure-img img-thumbnail padding-large" alt=""/>
-                        <figcaption class="figure-caption py-3">Add a caption to enhance the meaning of this image.</figcaption>
+                        <img src="/web/image/website.s_picture_default_image" class="figure-img img-thumbnail mx-auto padding-large" style="width: 75%;" alt=""/>
```

Comment:
Is the offset that bad? If yes, you could remove it, keep col-lg-10 and use justify-content-center on the row. That way resizing the col to control the image size will keep it centered.

Reply:
@sbuhl Is the column offset in the Picture snippet something very problematic?
@qsm-odoo I like your idea, but then, the behavior is different when you resize the column using the right arrow.

And what if I drop the commit about adding the 75% option to the editor and just keep this commit?

Reply:
> but then, the behavior is different when you resize the column using the right arrow.
Not sure it is a problem for me, @sbuhl ?
Removing the 75% option but keeping 75% on the image is a solution too but not ideal as if the user changes the width he has no way to go back to 75%.

Reply:
Dropped for now. The offset is not that bad.